components/availabilities/DocAvailability/AvailabilityList.tsx

```diff
@@ -0,0 +1,26 @@
+import { Availability } from "../../../types/Availabilities";
+import styles from "../../../styles/Availabilities.module.css";
+import AvailabilityCard from "./AvailabilityCard";
+import React from "react";
+
+interface AvailabilityListProps {
+  availabilities: Availability[];
+  onBook: (date: string) => void;
+}
+
+const AvailabilitiesList = ({
+  availabilities,
+  onBook,
+}: AvailabilityListProps) => (
+  <div className={styles.availabilityList}>
+    {availabilities.map((availability: Availability) => (
```

Comment:
There is no need to type `availability` here

```suggestion
    {availabilities.map((availability) => (
```